### PR TITLE
Ricardo barbosa

### PR DESCRIPTION
**Pull request by BarbosaRicardo**

### CODE REVIEW

**Review by lindseyreynolds44 — APPROVED** [review]
Looks good so far!

**Review by maximanss — APPROVED** [review]

File: Card.java [hunk 1]
```diff
@@ -49,39 +50,60 @@ public boolean getErrorFlag()
...
-      {
-          //if statement with errorFlag as parameter
-          //else return value + 'of' + suit string 
-      }
+    //Valid card data returned 
+    public String toString()
+    {
+        if(errorFlag)
+            return "Invalid Card";
+        return value + "of" + suit;
```
Comment (lindseyreynolds44) [inline Card.java]: Did this print out correctly? I think we may just need to add spaces before and after "of"

Comment (BarbosaRicardo) [reply Card.java]: Yes. It has been fixed and pushed back into my branch

**Comment by maximanss** [comment]
Ricardo,

I need a Copy Constructor in the Card class as follows:
    // Copy Constructor
    public Card (Card otherCard)
    {
      set (otherCard.value, otherCard.suit);
    }

File: src/Card.java [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    private char value;
+    private Suit suit;
+
+    //Checks for illegal card data
+    private boolean errorFlag;
+
+    //Constructors
+    public Card(char value, Suit suit)
+    {
+        set(value, suit);
```
Comment (thedandano) [inline src/Card.java:29]: Cleaver, I never though about using a method to instantiate the values.

**Review by thedandano — APPROVED** [review]
Looks good

**Pull request closed without merging**